contracts/axelarnet-gateway/src/state.rs

```diff
@@ -53,286 +30,297 @@ pub enum Error {
     MessageAlreadyExists(CrossChainId),
 }
 
+#[cw_serde]
+pub(crate) struct Config {
+    pub chain_name: ChainName,
+    pub router: Addr,
+}
+
+#[cw_serde]
+pub enum ExecutableMessage {
+    NotExecuted(Message),
```

Comment:
```suggestion
    Approved(Message),
```

Reply:
I don't really like `Approved`, because it's not immediately clear what it means, but I'll follow the team quorum on this